content/docs/dvclive/how-it-works.md

```diff
@@ -40,7 +41,7 @@ with Live(save_dvc_exp=True) as live:
     for i in range(EPOCHS):
         live.log_metric("metric", i + random.random())
         live.log_metric("nested/metric", i + random.random())
-        live.log_image("img.png", Image.new("RGB", (50, 50), (i, i, i)))
+        live.log_image(f"img/{live.step}.png", Image.new("RGB", (50, 50), (i, i, i)))
         Path("model.pt").write_text(str(random.random()))
```

Comment:
@daavoo Here's where the import statement comes from (sorry that it's unrelated)

Reply:
I see